src/components/room-invitation-creation-modal.js

```diff
@@ -46,16 +77,29 @@ function RoomInvitationCreationModal({ isVisible, onOk, onCancel, roomId }) {
 
   return (
     <Modal
-      title={t('newRoomInvitation')}
+      title={t('sendRoomInvitations')}
       onCancel={handleModalCancel}
       onOk={handleModalOk}
-      maskClosable={false}
       visible={isVisible}
       okButtonProps={{ loading: isLoading }}
       forceRender
       >
-      <Form form={form} name="new-room-invitation-form" initialValues={initialFormValues} onFinish={handleFormFinish} layout="vertical">
-        <EmailFormItem name="email" />
+      <Form
+        form={form}
+        layout="vertical"
+        name="send-room-invitations-form"
```

Comment:
Tiny thing: does the form need to have `name` set?

Reply:
I assumed as it was there before, but I just tried and I don't think it's necessary, so I removed it.